kotsadm/pkg/automation/automation.go

```diff
@@ -213,6 +393,9 @@ func getAirgapData(clientset kubernetes.Interface, license *kotsv1beta1.License)
 	result := map[string][]byte{}
 	for _, configMap := range configMaps.Items {
 		for key, value := range configMap.Data {
+			if key == "wait-for-airgap-app" { // do this better
```

Comment:
Should we now? Would it be less brittle if we just assumed that the absence of the key/configmap means "wait"?

Reply:
this is just skipping a bool value because bse64 decode works on those too....  it's file, file, file, bool!